### DISCUSSION TITLE
V1.0.9, I still don't care about cookies Latest

### DISCUSSION THREAD

**Discussion by OhMyGuus** [OhMyGuus/I-Still-Dont-Care-About-Cookies#423] [Announcements]
# I still don't care about cookies  V1.0.9

## (RECOMMENDED) Installation using official chrome & Firefox store
- [**Firefox** browser add-ons](https://addons.mozilla.org/en-US/firefox/addon/istilldontcareaboutcookies/
)
- [**Chrome** Web Store](https://chrome.google.com/webstore/detail/i-still-dont-care-about-c/edibdbjcniadpccecjdfdjjppcpchdlm)
- [**Edge** Add-ons](https://microsoftedge.microsoft.com/addons/detail/i-still-dont-care-about-/kkacdgacpkediooahopgcbdahlpipheh)

## Manual installation guide

- [Manual installation guide for Firefox](https://github.com/OhMyGuus/I-Dont-Care-About-Cookies/wiki/Firefox-installation-guide)
- [Manual installation guide for Chrome](https://github.com/OhMyGuus/I-Dont-Care-About-Cookies/wiki/Chrome-installation-guide)

## Contributors & website reporters
Thanks a lot to the contributors who are willing to contribute to the project by either reporting websites or by willing to implement websites into the extension 👍 
(Especially thanks a lot @appeasementPolitik for contributing a lot this release 😄 )

## Contact
_If u need to contact me create a GH issue or message me on discord 
Discord: ThaGuus#2140_
Email: Info[AT]istilldontcareaboutcookies.com
Github: [Issues](https://github.com/OhMyGuus/I-Dont-Care-About-Cookies/issues/new)

## Main changes
- Fixed issues with Manifest V3 service worker, things like the report button gone, whitelists being ignored etc.
- Added support for a lot of reported websites. 
- Changed the iframe event again and hopefully it should detect it better now

## Changelog / merged PR's
* Fix beruby.com by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/284
* Fix 123inkt.nl by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/290
* Fix fakenewsfrance.com by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/292
* Fix tourisme-lodevois-larzac.fr by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/294
* Fix drouwenerzand.nl by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/295
* Fix stihl.de by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/296
* Fix soundsmag.com by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/297
* Fix arche.net by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/304
* Fix scaleway.com by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/305
* Fix fresh-pool.de by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/306
* Fix/new websites 9 by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/317
* Fix swiss.com by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/335
* Fix befirst.info by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/336
* Fix cleveroad.com by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/337
* Fix etiskarna.si by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/338
* Fix solvimus.de by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/339
* Fix archerresourcing.co.uk by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/340
* Fix dlive.tv by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/344
* Fix honda.co.za by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/345
* Fix roland-rechtsschutz.de by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/347
* Fix plateuptools.com by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/348
* Changed to recreateTabList on extension reload by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/67
* Fix hd.digital by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/371
* Fix kauppakeskuswilla.fi by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/372
* Fix candis.io by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/373
* Fix bev.gv.at by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/374
* Fix pravniprostor.cz by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/375
* Fix marcos.com by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/392
* Fix cteasy.com by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/393
* Fix avax.network by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/394
* Fix lvvwd.com by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/395
* Fix parkside-interactive.com by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/396
* Fix beste-badstudios.de by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/397
* Fix ecobikecompany.com by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/398
* Fix snwa.com by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/399
* Fix aviyel.com by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/355
* Fixed reported websites - 9 by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/352
* Fixed support for 123inkt.nl by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/409
* Fix testo.com by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/413
* Fix seekube.com by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/414
* Fixes for Manifest v3 by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/363
* Fix/reported websites 10 by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/418
* Fixed insertcss/js path resolution for Firefox by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/419
* Run prettier & lint by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/420
* Pushed version number to 1.0.9 by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/421

**Full Changelog**: https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/compare/V1.0.8...v1.0.9

<hr /><em>This discussion was created from the release <a href='https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/releases/tag/v1.0.9'>V1.0.9, I still don't care about cookies Latest</a>.</em>

**Comment by Doublejay** [comment]
thanks for keeping this going !
